### ISSUE TITLE
Compiler warning: unreferenced local variable

### ISSUE THREAD

**Issue by sphoto** [osmcode/osmium-tool#203]
While building osmium-tool (using VS2015 on Win10) some warnings occur, due to libosmium commit [efd7e07](https://github.com/osmcode/libosmium/commit/efd7e07994e0d77c7163b44fe2a8798bec3d7a63#diff-3d4ff8d817d94433293af6300d1d3d9b42f78569ccf464cb066fecb65077ab6b) (timer.hpp)

```
D:\projects\osmium\20201206\libosmium\include\osmium/area/detail/basic_assembler.hpp(1110): warning C4101: 'timer_simpl
e_case': unreferenced local variable [d:\projects\osmium\20201206\osmium-tool\build\src\osmium.vcxproj]
D:\projects\osmium\20201206\libosmium\include\osmium/area/detail/basic_assembler.hpp(1028): warning C4101: 'timer_sort'
: unreferenced local variable [d:\projects\osmium\20201206\osmium-tool\build\src\osmium.vcxproj]
D:\projects\osmium\20201206\libosmium\include\osmium/area/detail/basic_assembler.hpp(1111): warning C4101: 'timer_compl
ex_case': unreferenced local variable [d:\projects\osmium\20201206\osmium-tool\build\src\osmium.vcxproj]
D:\projects\osmium\20201206\libosmium\include\osmium/area/detail/basic_assembler.hpp(1079): warning C4101: 'timer_split
': unreferenced local variable [d:\projects\osmium\20201206\osmium-tool\build\src\osmium.vcxproj]
D:\projects\osmium\20201206\libosmium\include\osmium/area/detail/basic_assembler.hpp(1035): warning C4101: 'timer_dupl'
: unreferenced local variable [d:\projects\osmium\20201206\osmium-tool\build\src\osmium.vcxproj]
D:\projects\osmium\20201206\libosmium\include\osmium/area/detail/basic_assembler.hpp(1059): warning C4101: 'timer_inter
section': unreferenced local variable [d:\projects\osmium\20201206\osmium-tool\build\src\osmium.vcxproj]
D:\projects\osmium\20201206\libosmium\include\osmium/area/detail/basic_assembler.hpp(1071): warning C4101: 'timer_locat
ions_list': unreferenced local variable [d:\projects\osmium\20201206\osmium-tool\build\src\osmium.vcxproj]
D:\projects\osmium\20201206\libosmium\include\osmium/area/detail/basic_assembler.hpp(1148): warning C4101: 'timer_roles
': unreferenced local variable [d:\projects\osmium\20201206\osmium-tool\build\src\osmium.vcxproj]
```

**Comment by joto** [comment]
Thanks for the report. I believe this has been fixed in libosmium 5e72f050 .

**Comment by sphoto** [comment]
Warnings are gone now, thanks!
